editor/editor_settings.cpp

```diff
@@ -1144,6 +1149,38 @@ void EditorSettings::setup_language() {
 	load_extractable_translations(lang);
 }
 
+String EditorSettings::get_online_docs_url() {
+	String language = EditorSettings::get_singleton()->get_setting("interface/editor/editor_language");
+	bool override = EditorSettings::get_singleton()->get_setting("interface/editor/override_online_docs_language");
+	String override_language = EditorSettings::get_singleton()->get_setting("interface/editor/online_docs_language");
+
+	Vector<String> doc_languages;
```

Comment:
What is the right way to store a static list like this? Note that this is also duplicated as a comma separated list in a string on line 401.